vizro-core/examples/_dev/app.py

```diff
@@ -1,23 +1,53 @@
-"""Rough example used by developers."""
-
 import vizro.models as vm
 import vizro.plotly.express as px
 from vizro import Vizro
+from vizro.actions import export_data
+from vizro.tables import dash_data_table
 
-df = px.data.iris()
+df = px.data.gapminder()
 
-page = vm.Page(
-    title="Page Title",
+table_and_container = vm.Page(
+    title="Table and Container",
     components=[
-        vm.Graph(figure=px.scatter(df, x="sepal_length", y="petal_width", color="species")),
-        vm.Graph(figure=px.histogram(df, x="sepal_width", color="species")),
-    ],
-    controls=[
-        vm.Filter(column="species"),
+        vm.Container(
+            title="Container w/ Table",
```

Comment:
Will be reverted - just for demo purposes

Reply:
That would also be fine for me 👍 I thought it would be just easier to delete the short default example for you guys rather than the longer dev examples I usually add here 😅 Fine either way to be honest